crates/iceberg/src/spec/manifest.rs

```diff
@@ -1182,6 +1182,12 @@ impl ManifestEntry {
     pub fn data_file(&self) -> &DataFile {
         &self.data_file
     }
+
+    /// get file sequence number
```

Comment:
```suggestion
    /// File sequence number indicating when the file was added. Inherited when null and status is 1 (added).
```

from https://iceberg.apache.org/spec/#manifests